lib/ecstatic/showdir.js

```diff
@@ -108,6 +109,8 @@ module.exports = function (opts, stat) {
             '  <head>',
             '    <meta charset="utf-8">',
             '    <meta name="viewport" content="width=device-width">',
+            '    <link href="/_ecstatic-assets/ecstatic.css" rel="stylesheet" type="text/css" />',
```

Comment:
An alternative to feature-flagging all CSS: embed it directly. That could be cool.